package.json

```diff
@@ -32,12 +49,13 @@
   "license": "MIT",
   "devDependencies": {
     "add-event-handler": "^1.0.0",
-    "babel": "^5.8.21",
+    "babel-plugin-transform-class-properties": "^6.3.13",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-register": "^6.3.13",
```

Comment:
I like testing the source. I know, it gets transpiled on the fly, but for some reason, it seems more appropriate to me. And that is what you do in [this example project](https://github.com/rollup/rollup-starter-project#testindex_testjs), right?